deform/tests/test_schema.py

```diff
@@ -20,6 +20,22 @@ def _makeOne(self):
 
         return FileData()
 
+    def test_deferred_csrf_value(self):
+        import colander
+        from deform.schema import CSRFSchema
+        from unittest.mock import MagicMock
+        from deform.schema import deferred_csrf_value
+
+        kw = {"request": MagicMock(name="request")}
+        kw["request"].session = MagicMock(name="session")
+        kw["request"].session.get_csrf_token = MagicMock(
+            return_value=colander.null)
+
+        node = DummySchemaNode()
+        new_schema = CSRFSchema()
+        new_schema.default = deferred_csrf_value(node, kw)
+        self.assertEqual(new_schema.default, colander.null)
+
```

Comment:
Would it be better to use an arbitrary value for the `return_value`? For example:

```
MOCK_CSRF_TOKEN = "SteveHasNoIdeaHowToFixThis"
...
            return_value=MOCK_CSRF_TOKEN)
...
            self.assertEqual(new_schema.default, MOCK_CSRF_TOKEN)
```

That would save an import of colander.